ra2ce/analysis/analysis_config_data/analysis_config_data.py

```diff
@@ -98,7 +98,6 @@ class AnalysisSectionLosses(AnalysisSectionBase):
     production_loss_per_capita_per_day: float = math.nan
     part_of_day: PartOfDayEnum = field(default_factory=lambda: PartOfDayEnum.DAY)
     performance: str = "diff_time"  # "diff_time" or "diff_length" relates to the used criticality metric
-    hours_per_day: float = 24
```

Comment:
I suggest to keep hours_per_day but not use it. It allows us later to make use of it. For instance it is relevant if we have traffic data for 3 hours of the morning rush hour.